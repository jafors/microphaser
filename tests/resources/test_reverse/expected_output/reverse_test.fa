>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>0b4e37d09934bdeR
acAgggacAggggccgggaccgggacc
>0bf5651d15c846cR
acAgggactggggccgggaccgggaca
>5829bfd284f0cb5R
gggacAgggacAggggccgggaccggg
>683eeee967ad8d0R
gggacAgggactggggccgggaccggg
>4087657e230f3a5R
accgggacAgggacAggggccgggacc
>7023ad3880e2d57R
accgggacAgggactggggccgggacc
>8c3434d759140efR
gggaccgggacAgggactggggccggg
>ae280e232d0582aR
gggaccgggacAgggacAggggccggg
>6f5b9cce754f8f2R
gggaccgggacAgggactggggccggg
>561f80087272f47R
gccgggaccgggacAgggactggggcc
>e1b098d6a1588e2R
gccgggaccgggacAgggacAggggcc
>e9da669b9cab341R
ggAgccgggaccgggacAgggactggg
>4e047277cdeb322R
ggggccgggaccgggacAgggacAggg
>fc553bd6250d877R
actggAgccgggaccgggacAgggact
>f561469dc0d1bbdR
actggggccgggaccgggacAgggacA
>8aaf4c7d467b4f4R
gggactggggccgggaccgggacAggg
>cf58ccde40a93c6R
gggactggAgccgggaccgggacAggg
>18fa98f4b970791R
accgggactggggccgggaccgggacA
>f7a0e246701c5ddR
acAgggactggggccgggaccgggacA
>d6d3ec26afc54dbR
acAgggactggAgccgggaccgggacA
