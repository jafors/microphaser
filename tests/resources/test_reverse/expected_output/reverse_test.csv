id,transcript,gene_id,gene_name,offset,freq,nvar,nsomatic,strand
62cb2647a87e5dfR,ENST00000314675,ENSG00000158062,UBXN11,26300986,0.4672897196261682,0,0,Reverse
a324f17d17b218dR,ENST00000314675,ENSG00000158062,UBXN11,26300986,0.5327102803738317,1,0,Reverse
6fb08ec7c7f47bcR,ENST00000314675,ENSG00000158062,UBXN11,26300983,0.48623853211009177,0,0,Reverse
c258647e023393eR,ENST00000314675,ENSG00000158062,UBXN11,26300983,0.5137614678899083,1,0,Reverse
694cc3c630d8067R,ENST00000314675,ENSG00000158062,UBXN11,26300980,0.49122807017543857,0,0,Reverse
172c41a2e6e2cb7R,ENST00000314675,ENSG00000158062,UBXN11,26300980,0.5087719298245614,1,0,Reverse
39562cfb831a701R,ENST00000314675,ENSG00000158062,UBXN11,26300977,0.4957983193277311,0,0,Reverse
b5f424dc651bf48R,ENST00000314675,ENSG00000158062,UBXN11,26300977,0.5042016806722689,1,0,Reverse
0d1064b66ef872eR,ENST00000314675,ENSG00000158062,UBXN11,26300974,0.4959349593495935,0,0,Reverse
965e45175515136R,ENST00000314675,ENSG00000158062,UBXN11,26300974,0.5040650406504065,1,0,Reverse
9dfd4a2a3e06a03R,ENST00000314675,ENSG00000158062,UBXN11,26300971,0.5161290322580645,0,0,Reverse
08add1803a50f62R,ENST00000314675,ENSG00000158062,UBXN11,26300971,0.4838709677419355,1,0,Reverse
1890ce6eb553e2eR,ENST00000314675,ENSG00000158062,UBXN11,26300968,0.5206611570247934,0,0,Reverse
223bd8fb09288ccR,ENST00000314675,ENSG00000158062,UBXN11,26300968,0.4793388429752066,1,0,Reverse
bbcca5e5926c6b8R,ENST00000314675,ENSG00000158062,UBXN11,26300965,0.512396694214876,0,0,Reverse
732c0f3ff28e5c0R,ENST00000314675,ENSG00000158062,UBXN11,26300965,0.48760330578512395,1,0,Reverse
4b3b6ad8c866d79R,ENST00000314675,ENSG00000158062,UBXN11,26300962,0.47540983606557374,0,0,Reverse
58c2ffd43739c73R,ENST00000314675,ENSG00000158062,UBXN11,26300962,0.5245901639344263,1,0,Reverse
c9376de4a4eeb36R,ENST00000314675,ENSG00000158062,UBXN11,26285966,0.7285714285714285,0,0,Reverse
1c060adfd690c42R,ENST00000314675,ENSG00000158062,UBXN11,26285966,0.2714285714285714,1,0,Reverse
70f4032847e8b9cR,ENST00000314675,ENSG00000158062,UBXN11,26285963,0.7123287671232876,0,0,Reverse
8584b7aeb9d2f26R,ENST00000314675,ENSG00000158062,UBXN11,26285963,0.2876712328767123,1,0,Reverse
89e88d210eb44afR,ENST00000314675,ENSG00000158062,UBXN11,26285960,0.7123287671232876,0,0,Reverse
a5528732feff657R,ENST00000314675,ENSG00000158062,UBXN11,26285960,0.2876712328767123,1,0,Reverse
5467b8035befff8R,ENST00000314675,ENSG00000158062,UBXN11,26285957,0.7236842105263158,0,0,Reverse
33bdff14891109fR,ENST00000314675,ENSG00000158062,UBXN11,26285957,0.27631578947368424,1,0,Reverse
20ad676ed0eeb1fR,ENST00000314675,ENSG00000158062,UBXN11,26285954,0.7272727272727273,0,0,Reverse
1e6e5cfb1cdc9deR,ENST00000314675,ENSG00000158062,UBXN11,26285954,0.2727272727272727,1,0,Reverse
63b7186258fc02dR,ENST00000314675,ENSG00000158062,UBXN11,26285951,0.7215189873417721,0,0,Reverse
86aae2a3ec5fcadR,ENST00000314675,ENSG00000158062,UBXN11,26285951,0.27848101265822783,1,0,Reverse
3f7096abd8bf649R,ENST00000314675,ENSG00000158062,UBXN11,26285948,0.7012987012987013,0,0,Reverse
caa2a4b00ce540eR,ENST00000314675,ENSG00000158062,UBXN11,26285948,0.2987012987012987,1,0,Reverse
caf61cdec0e89e5R,ENST00000314675,ENSG00000158062,UBXN11,26285945,0.7012987012987013,0,0,Reverse
5aaa3507d1d1348R,ENST00000314675,ENSG00000158062,UBXN11,26285945,0.2987012987012987,1,0,Reverse
36dee4a50c2493eR,ENST00000314675,ENSG00000158062,UBXN11,26285942,0.6973684210526315,0,0,Reverse
c204eec32d41624R,ENST00000314675,ENSG00000158062,UBXN11,26285942,0.3026315789473684,1,0,Reverse
742163ac1cebc12R,ENST00000314675,ENSG00000158062,UBXN11,26284398,1,1,0,Reverse
fd764a5345f7473R,ENST00000314675,ENSG00000158062,UBXN11,26284395,1,1,0,Reverse
a55d1f2587eae5cR,ENST00000314675,ENSG00000158062,UBXN11,26284392,1,1,0,Reverse
ae580a513507433R,ENST00000314675,ENSG00000158062,UBXN11,26284389,1,1,0,Reverse
946de2980abc10fR,ENST00000314675,ENSG00000158062,UBXN11,26284386,1,1,0,Reverse
4d2f7903355cae3R,ENST00000314675,ENSG00000158062,UBXN11,26284383,1,1,0,Reverse
8a68b9eec214f8bR,ENST00000314675,ENSG00000158062,UBXN11,26284380,1,1,0,Reverse
56239f5beba4a30R,ENST00000314675,ENSG00000158062,UBXN11,26284377,1,1,0,Reverse
c8cd1f33758a3fcR,ENST00000314675,ENSG00000158062,UBXN11,26284374,1,1,0,Reverse
577e9ef1d73e376R,ENST00000314675,ENSG00000158062,UBXN11,26282761,0.46875,0,0,Reverse
6744a2c2d41f673R,ENST00000314675,ENSG00000158062,UBXN11,26282761,0.53125,1,0,Reverse
8e681aa30658d5aR,ENST00000314675,ENSG00000158062,UBXN11,26282758,0.47619047619047616,0,0,Reverse
871c2db59be2e24R,ENST00000314675,ENSG00000158062,UBXN11,26282758,0.5238095238095238,1,0,Reverse
fb16dc2abaad395R,ENST00000314675,ENSG00000158062,UBXN11,26282755,0.47761194029850745,0,0,Reverse
a50701944aeb40fR,ENST00000314675,ENSG00000158062,UBXN11,26282755,0.5223880597014925,1,0,Reverse
e263dcf1cffd1ecR,ENST00000314675,ENSG00000158062,UBXN11,26282752,0.5070422535211268,0,0,Reverse
6cbeb500d27ab0cR,ENST00000314675,ENSG00000158062,UBXN11,26282752,0.49295774647887325,1,0,Reverse
8870fc955acfa6aR,ENST00000314675,ENSG00000158062,UBXN11,26282749,0.4794520547945205,0,0,Reverse
9f825be514211d0R,ENST00000314675,ENSG00000158062,UBXN11,26282749,0.5205479452054794,1,0,Reverse
2a0306ddf027cf8R,ENST00000314675,ENSG00000158062,UBXN11,26282746,0.4675324675324675,0,0,Reverse
380d4855b31900cR,ENST00000314675,ENSG00000158062,UBXN11,26282746,0.5324675324675324,1,0,Reverse
8afcef1a5734f6aR,ENST00000314675,ENSG00000158062,UBXN11,26282743,0.4567901234567901,0,0,Reverse
47689d704d16372R,ENST00000314675,ENSG00000158062,UBXN11,26282743,0.5432098765432098,1,0,Reverse
e45b398cee8397eR,ENST00000314675,ENSG00000158062,UBXN11,26282385,0.8529411764705882,0,0,Reverse
aee7752445f7ac7R,ENST00000314675,ENSG00000158062,UBXN11,26282385,0.14705882352941177,1,0,Reverse
06e4d6fa2d0a92cR,ENST00000314675,ENSG00000158062,UBXN11,26282382,0.8484848484848485,0,0,Reverse
e153e44397d0225R,ENST00000314675,ENSG00000158062,UBXN11,26282382,0.15151515151515152,1,0,Reverse
086ad8ff06c33deR,ENST00000314675,ENSG00000158062,UBXN11,26282379,0.5161290322580645,0,0,Reverse
bc7fb48dd3846c6R,ENST00000314675,ENSG00000158062,UBXN11,26282379,0.3225806451612903,1,0,Reverse
086ad8ff06c33deR,ENST00000314675,ENSG00000158062,UBXN11,26282379,0.16129032258064516,0,0,Reverse
0f1f6e85f76e173R,ENST00000314675,ENSG00000158062,UBXN11,26282376,0.5161290322580645,0,0,Reverse
7e52ccba441a7b0R,ENST00000314675,ENSG00000158062,UBXN11,26282376,0.3548387096774194,1,0,Reverse
0f1f6e85f76e173R,ENST00000314675,ENSG00000158062,UBXN11,26282376,0.12903225806451613,0,0,Reverse
1e4997b22ab45d2R,ENST00000314675,ENSG00000158062,UBXN11,26282373,0.4827586206896552,0,0,Reverse
1e4997b22ab45d2R,ENST00000314675,ENSG00000158062,UBXN11,26282373,0.3448275862068966,0,0,Reverse
c3522362d359e4bR,ENST00000314675,ENSG00000158062,UBXN11,26282373,0.034482758620689655,1,0,Reverse
c3522362d359e4bR,ENST00000314675,ENSG00000158062,UBXN11,26282373,0.13793103448275862,1,0,Reverse
9be96014cc873e3R,ENST00000314675,ENSG00000158062,UBXN11,26282370,0.4642857142857143,0,0,Reverse
9be96014cc873e3R,ENST00000314675,ENSG00000158062,UBXN11,26282370,0.35714285714285715,0,0,Reverse
03a8d16f9377bd0R,ENST00000314675,ENSG00000158062,UBXN11,26282370,0.03571428571428571,1,0,Reverse
03a8d16f9377bd0R,ENST00000314675,ENSG00000158062,UBXN11,26282370,0.14285714285714285,1,0,Reverse
5935263355d21baR,ENST00000314675,ENSG00000158062,UBXN11,26282367,0.4,0,0,Reverse
5935263355d21baR,ENST00000314675,ENSG00000158062,UBXN11,26282367,0.4,0,0,Reverse
544a0c9a48cd79fR,ENST00000314675,ENSG00000158062,UBXN11,26282367,0.04,1,0,Reverse
544a0c9a48cd79fR,ENST00000314675,ENSG00000158062,UBXN11,26282367,0.16,1,0,Reverse
fd0f95bc205cdf2R,ENST00000314675,ENSG00000158062,UBXN11,26282364,0.36363636363636365,0,0,Reverse
6ff07d3f62e4a2cR,ENST00000314675,ENSG00000158062,UBXN11,26282364,0.09090909090909091,1,0,Reverse
fd0f95bc205cdf2R,ENST00000314675,ENSG00000158062,UBXN11,26282364,0.36363636363636365,0,0,Reverse
6ff07d3f62e4a2cR,ENST00000314675,ENSG00000158062,UBXN11,26282364,0.045454545454545456,1,0,Reverse
6ff07d3f62e4a2cR,ENST00000314675,ENSG00000158062,UBXN11,26282364,0.13636363636363635,1,0,Reverse
f7a8162bb062070R,ENST00000314675,ENSG00000158062,UBXN11,26282361,0.4,0,0,Reverse
8c910cb425ca167R,ENST00000314675,ENSG00000158062,UBXN11,26282361,0.13333333333333333,1,0,Reverse
f7a8162bb062070R,ENST00000314675,ENSG00000158062,UBXN11,26282361,0.4,0,0,Reverse
8c910cb425ca167R,ENST00000314675,ENSG00000158062,UBXN11,26282361,0.06666666666666667,1,0,Reverse
0fd5f2de5cb129bR,ENST00000314675,ENSG00000158062,UBXN11,26282358,0.375,1,0,Reverse
0fd5f2de5cb129bR,ENST00000314675,ENSG00000158062,UBXN11,26282358,0.1875,1,0,Reverse
4ed34e6b57bf437R,ENST00000314675,ENSG00000158062,UBXN11,26282358,0.375,0,0,Reverse
4ed34e6b57bf437R,ENST00000314675,ENSG00000158062,UBXN11,26282358,0.0625,0,0,Reverse
920ad7f4c1d6d7cR,ENST00000314675,ENSG00000158062,UBXN11,26282355,0.4,1,0,Reverse
920ad7f4c1d6d7cR,ENST00000314675,ENSG00000158062,UBXN11,26282355,0.2,1,0,Reverse
384d5ff5188d8ceR,ENST00000314675,ENSG00000158062,UBXN11,26282355,0.3333333333333333,0,0,Reverse
384d5ff5188d8ceR,ENST00000314675,ENSG00000158062,UBXN11,26282355,0.06666666666666667,0,0,Reverse
35401280f1deda7R,ENST00000314675,ENSG00000158062,UBXN11,26282352,0.38461538461538464,0,0,Reverse
16d5d645d9f4205R,ENST00000314675,ENSG00000158062,UBXN11,26282352,0.23076923076923078,1,0,Reverse
35401280f1deda7R,ENST00000314675,ENSG00000158062,UBXN11,26282352,0.15384615384615385,0,0,Reverse
16d5d645d9f4205R,ENST00000314675,ENSG00000158062,UBXN11,26282352,0.23076923076923078,1,0,Reverse
dfe92347736da03R,ENST00000314675,ENSG00000158062,UBXN11,26282349,0.38461538461538464,0,0,Reverse
dfe92347736da03R,ENST00000314675,ENSG00000158062,UBXN11,26282349,0.07692307692307693,0,0,Reverse
dfe92347736da03R,ENST00000314675,ENSG00000158062,UBXN11,26282349,0.15384615384615385,0,0,Reverse
dfe92347736da03R,ENST00000314675,ENSG00000158062,UBXN11,26282349,0.07692307692307693,0,0,Reverse
dfe92347736da03R,ENST00000314675,ENSG00000158062,UBXN11,26282349,0.07692307692307693,0,0,Reverse
dfe92347736da03R,ENST00000314675,ENSG00000158062,UBXN11,26282349,0.23076923076923078,0,0,Reverse
560715910a2259fR,ENST00000314675,ENSG00000158062,UBXN11,26282346,0.07692307692307693,0,0,Reverse
560715910a2259fR,ENST00000314675,ENSG00000158062,UBXN11,26282346,0.46153846153846156,0,0,Reverse
560715910a2259fR,ENST00000314675,ENSG00000158062,UBXN11,26282346,0.07692307692307693,0,0,Reverse
560715910a2259fR,ENST00000314675,ENSG00000158062,UBXN11,26282346,0.38461538461538464,0,0,Reverse
f762b9aff44b7baR,ENST00000314675,ENSG00000158062,UBXN11,26282343,0.1,0,0,Reverse
f762b9aff44b7baR,ENST00000314675,ENSG00000158062,UBXN11,26282343,0.3,0,0,Reverse
f762b9aff44b7baR,ENST00000314675,ENSG00000158062,UBXN11,26282343,0.1,0,0,Reverse
f762b9aff44b7baR,ENST00000314675,ENSG00000158062,UBXN11,26282343,0.5,0,0,Reverse
ed4c5d4ddac6d98R,ENST00000314675,ENSG00000158062,UBXN11,26282340,0.1,0,0,Reverse
ed4c5d4ddac6d98R,ENST00000314675,ENSG00000158062,UBXN11,26282340,0.3,0,0,Reverse
ed4c5d4ddac6d98R,ENST00000314675,ENSG00000158062,UBXN11,26282340,0.1,0,0,Reverse
ed4c5d4ddac6d98R,ENST00000314675,ENSG00000158062,UBXN11,26282340,0.5,0,0,Reverse
076975fa347c701R,ENST00000314675,ENSG00000158062,UBXN11,26282337,0.1,0,0,Reverse
076975fa347c701R,ENST00000314675,ENSG00000158062,UBXN11,26282337,0.3,0,0,Reverse
076975fa347c701R,ENST00000314675,ENSG00000158062,UBXN11,26282337,0.1,0,0,Reverse
076975fa347c701R,ENST00000314675,ENSG00000158062,UBXN11,26282337,0.5,0,0,Reverse
828e75a3e7deeeaR,ENST00000314675,ENSG00000158062,UBXN11,26282334,0.25,0,0,Reverse
828e75a3e7deeeaR,ENST00000314675,ENSG00000158062,UBXN11,26282334,0.25,0,0,Reverse
828e75a3e7deeeaR,ENST00000314675,ENSG00000158062,UBXN11,26282334,0.08333333333333333,0,0,Reverse
828e75a3e7deeeaR,ENST00000314675,ENSG00000158062,UBXN11,26282334,0.4166666666666667,0,0,Reverse
bdbf32be8271a0cR,ENST00000314675,ENSG00000158062,UBXN11,26282331,0.5,0,0,Reverse
4ed424b0372a61eR,ENST00000314675,ENSG00000158062,UBXN11,26282331,0.5,1,0,Reverse
7b456d8781464bdR,ENST00000314675,ENSG00000158062,UBXN11,26282328,0.5555555555555556,0,0,Reverse
168ebadb6cb6e9dR,ENST00000314675,ENSG00000158062,UBXN11,26282328,0.4444444444444444,1,0,Reverse
45531e958a7133eR,ENST00000314675,ENSG00000158062,UBXN11,26282325,0.5555555555555556,0,0,Reverse
3f9f2ad69ae3ebbR,ENST00000314675,ENSG00000158062,UBXN11,26282325,0.4444444444444444,1,0,Reverse
